gnovm/pkg/gnolang/store.go

```diff
@@ -25,9 +25,6 @@ import (
 // cause writes to happen to the store, such as MemPackages to iavlstore.
 type PackageGetter func(pkgPath string, store Store) (*PackageNode, *PackageValue)
 
-// inject natives into a new or loaded package (value and node)
-type PackageInjector func(store Store, pn *PackageNode)
```

Comment:
can you give some tips why the store change is needed?

Reply:
This is part of the [native bindings masterplan](https://github.com/gnolang/gno/issues/814), which has been a long time coming...

Essentially, native bindings changed from using the PackageInjector to having a different system to declare "natively bound" functions (ie. foreign-function interface). This was carried out for pretty much all functions, except those that were using DefineGoNativeValue and `strconv`, which was an outlier due to being a good candidate to be moved entirely to standard libraries, which took longer than expected to do properly; but you helped happen thanks to #1775 :)

Eventually, after #1361, the goal is to only use native bindings as a way to have a FFI from Go into Gno. But that's still going to take another short while. Hopefully that explains it :)